lib/src/airship_utils.dart

```diff
@@ -0,0 +1,34 @@
+import "feature.dart";
+
+class AirshipUtils {
+  static List<Feature> parseFeatures(List<String> strings) {
+    var features = <Feature>[];
+    strings.forEach((element) {
+      switch (element) {
+        case "push": features.add(Feature.push); break;
+        case "analytics": features.add(Feature.analytics); break;
+        case "in_app_automation": features.add(Feature.inAppAutomation); break;
+        case "tags_and_attributes": features.add(Feature.tagsAndAttributes); break;
+        case "message_center": features.add(Feature.messageCenter); break;
+        case "contacts": features.add(Feature.contacts); break;
+        default: print("Invalid feature");
```

Comment:
can we do `print("Invalid feature: $element")` here? seems like it might come in handy